core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -133,7 +138,7 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
     ) {}
 
     async loadCustomBlocks(urls: string[]) {
-        return Promise.all(urls.map(async (url) => import(/* webpackIgnore: true */ url)));
+        return Promise.allSettled(urls.map(async (url) => import(/* webpackIgnore: true */ url)));
```

Comment:
Please type this, because right nos TSC is guessing: `PromiseSettledResult<any>[]`